File: BasicsWithJava/BlockchainApp/app/src/main/java/com/sumere/blockchainapp/service/CryptoAPI.java
package com.sumere.blockchainapp.service;

import com.sumere.blockchainapp.view.CryptoModel;

import java.util.List;

import io.reactivex.Observable;
import retrofit2.Call;
import retrofit2.http.GET;

public interface CryptoAPI {
    //GET, POST, UPDATE, DELETE
    @GET("atilsamancioglu/K21-JSONDataSet/master/crypto.json")
    Observable<List<CryptoModel>> getData();
    //Call<List<CryptoModel>> getData();
}

File: KotlinCryptoCrazy/app/src/main/java/com/sumere/kotlincryptocrazy/service/CryptoAPI.kt
package com.sumere.kotlincryptocrazy.service

import com.sumere.kotlincryptocrazy.model.CryptoModel
import retrofit2.Response
import retrofit2.http.GET

interface CryptoAPI {
    //https://raw.githubusercontent.com/atilsamancioglu/K21-JSONDataSet/master/crypto.json
    @GET("atilsamancioglu/K21-JSONDataSet/master/crypto.json")
    suspend fun getData(): Response<List<CryptoModel>>
}